Lavinia-api/Models/PartyVotes.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace LaviniaApi.Models
+{
+    // API v2
```

Comment:
It might have made more sense to namespace everything API V2 related instead of using comments.